### DISCUSSION TITLE
How can I get all findings/vulnerabilities in project

### DISCUSSION THREAD

**Discussion by Stamka** [pablosnt/rekono#230] [Help]
Hi. I create project and do scan, but I can see findings/vulnerabilities by target, how can I get all findings/vulnerabilities for all targets in project?

**Comment by pablosnt** [comment]
In that moment, it is not possible because the findings UI forces the users to choose a target. The reason is that depending on the findings, you would need to have the target context to understand them. For example, if you have two private networks as target, it's possible to have host findings with the same value and it would be confusing if you don't filter by target. However, I think that allowing to display findings from all project targets (and even all projects) could be a good new feature. I will consider that for the Rekono 2.0 version. Thank you for the proposal!

I know that it's not what you are looking for, but you could do it using the API Rest and [rekono-cli](https://github.com/pablosnt/rekono-cli).

**Reply by Stamka** [reply]
The problem is that I have 40-100 targets in project and after scanning I get 150 finding. In current implementation I need to manually change target then change finding type (path, host vulnerability) and I need to spend a lot of type for manually check all target and all findings instead of get full information about all targets in project.

For example in findings tab I want to see. Host 10.10.10.10, and all findings.

When you are planning to release rekono 2.0? If it not soon I can try to do this feature

**Reply by pablosnt** [reply]
Yes, I understand your problem, the feature of filtering findings as the user wants/needs should be included in Rekono.

To be honest, I expect to release Rekono 2.0 close to the year end. My optimistic estimation is the summer end, but I think I will need more time. If you want to contribute the feature, it will be welcomed, and it could be released in a 1.6.2 version. Thank you!

**Comment by pablosnt** [comment]
This issue has been fixed as part of the new version 2.0.0, where you can explore all the findings without mandatory filters.
